jwst/ramp_fitting/ramp_fit_step.py

```diff
@@ -231,11 +232,18 @@ def process(self, input):
             else:
                 input_model.int_times = None
 
+            ramp_fit_dq_flags = {
+                "DO_NOT_USE": dqflags.pixel["DO_NOT_USE"],
+                "JUMP_DET": dqflags.pixel["JUMP_DET"],
+                "SATURATED": dqflags.pixel["SATURATED"],
+                "NO_GAIN_VALUE": dqflags.pixel["NO_GAIN_VALUE"],
+                "UNRELIABLE_SLOPE": dqflags.pixel["UNRELIABLE_SLOPE"],
+            }
```

Comment:
I think originally some of these DQ flags were from `jwst.datamodels.dqflags.group` dict.

https://github.com/spacetelescope/jwst/blob/f8d9122b78dd4c4f34df5ca0199e0857116ed0a3/jwst/ramp_fitting/ramp_fit.py#L33-L34

And some from the pixel dict:

https://github.com/spacetelescope/jwst/blob/f8d9122b78dd4c4f34df5ca0199e0857116ed0a3/jwst/ramp_fitting/ramp_fit.py#L1547

Though it seems the group and pixel dq flags are identical.  (above code snippets from release 1.1.0)

Is there a reason to redefine them here in the calling code instead of just shuffling the full dictionary on to `ramp_fit` and letting it use them as it used them before?

Reply:
The `group` flags are defined from the `pixel` dictionary.  I simply created these test DQ flag dictionaries as a demonstration that the values could be defined to be anything, so long as they are consistent, as well as to show the required key words to pass to `ramp_fit`.

Reply:
I changed the call to `ramp_fit` to simply pass `dqflags.pixel`, rather than create a separate dictionary to pass.